db/migrations/20150327080540_add_cached_docker_image_to_droplets.rb

```diff
@@ -0,0 +1,5 @@
+Sequel.migration do
+  change do
+    add_column :droplets, :cached_docker_image, String
```

Comment:
If the cached_docker_image can ever exceed ~250 characters, it's better to make this a text field by adding `, text: true` after `String`.